### DISCUSSION TITLE
Localization for email template

### DISCUSSION THREAD

**Discussion by stf-alexander** [spatie/laravel-support-bubble#30] [Ideas]
Hi guys,

currenty I can't find any option to change the subject for the email template that is sent to "mail_to" receiver. It's default is `Support bubble message from` and there is also a line saying `left a new message using the chat bubble:`. How about making these string configurable via the lang files in `resources/lang/vendor/laravel-support-bubble/en/` :-) ?

Best regards,
Alexander

**Comment by freekmurze** [comment]
Currently that "left a new message using the chat bubble" isn't translatable indeed. I'd accept a PR that makes it translatable.
